dbt/README.md

```diff
@@ -235,3 +235,70 @@ This produces the output:
 
 This is failing because the `max_value` is set to `0.7`. If we change this value
 to 0.83, this test should now pass.
+
+#### `no_null_cols` tests
```

Comment:
I think this test is run on a very large number of columns currently. It might be a good candidate for auto-generation, or being turned into a table-level test that by default checks all columns, but allows certain columns to be excluded (since in the `etl_fast` outputs columns that were only reported in older historical versions of the data are expected to be entirely null in modern data)